cdc/puller/sorter/heap_sorter.go

```diff
@@ -159,7 +160,7 @@ func (h *heapSorter) flush(ctx context.Context, maxResolvedTs uint64) error {
 			return nil
 		}
 		oldHeap = h.heap
-		h.heap = make(sortHeap, 0, 65536)
+		h.heap = make(sortHeap, 0, sortHeapCapacity)
```

Comment:
Do we need to allocate a new slice for `heap`? Can we reuse the previous one, eg: `h.heap = h.heap[:0]`

Reply:
No. Flushing the heap to disk is done in a separate goroutine, so we need to keep the original underlying array intact until it's been flushed.